Doc: Anonymous events (#9497)
PostHog/posthog.com: contents/docs/data/persons.mdx

People in PostHog represent the users behind your events. You can view them in the [People tab](https://us.posthog.com/persons).

People have **person profiles** with [properties](/docs/product-analytics/person-properties). This enables you to do things like:

- Filter on person properties
- Create [cohorts](/docs/data/cohorts)
- Use person properties for targeting feature flags, A/B tests, and surveys.
- Track initial UTM values and referrers across anonymous and [identified](/docs/product-analytics/identify) users.
- Query the `persons` table using [SQL insights](/docs/product-analytics/sql)

## How to create person profiles

> When you capture your first [identified event](/docs/data/anonymous-vs-identified-events) for a user, it creates a **person profile** for them. Then, any future events captured will be attributed to them. You can also [set properties](/docs/product-analytics/person-properties) for the user.

Reviewer: ```suggestion
When you capture your first [identified event](/docs/data/anonymous-vs-identified-events) for a user, it creates a **person profile** for them. Then, any future events captured are attributed to this profile. You can also [set properties](/docs/product-analytics/person-properties) for the person.
```
